report.md

```diff
@@ -476,6 +477,30 @@ When writing `require` or `if` conditionals that check storage values, it is imp
 	        if (number > s_differentConditionals) {
 	```
 
+- Found in src/eth2/DepositContract.sol [Line: 97](tests/contract-playground/src/eth2/DepositContract.sol#L97)
```

Comment:
These are incorrect because they're in the same function. This detector should discount items within the same scope?

Reply:
Tracked in #117